src/components/Caption/index.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from "classnames";
+
+import styles from "./Caption.module.scss";
+
+export interface CaptionProps {
+  header?: JSX.Element;
+  title: JSX.Element;
+  className?: string;
+  color?: "green" | "sky" | "blue";
+  textSize?: "small" | "medium" | "costume";
```

Comment:
Costume, is that meant to be custom?

Reply:
I know.. ther is no consistency in terms of font size. It looks ugly. I will see what I can do